notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.                victim_df.groupby("adv_steps")
</pre><p>This group by is used in 3 different places so can probably just assign to a temp variable and reuse</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>